kuzminskiy_andrey/gen_graph.cpp

```diff
@@ -27,17 +25,13 @@ class Graph {
 
  private:
   bool has_vertex(VertexId id) const {
-    if (std::any_of(vertices_.begin(), vertices_.end(),
-                    [&id](Vertex v) { return v.id() == id; })) {
-      return true;
-    }
-
-    return false;
+    return std::any_of(vertices_.begin(), vertices_.end(),
+                       [id](Vertex v) { return v.id() == id; });
```

Comment:
Не примитивные данные лучше передавать по конст ссылке
```suggestion
                       [id](const Vertex& v) { return v.id() == id; });
```

Reply:
Не исправил